lib/cdo/shared_constants/pd/facilitator1819_application_constants.rb

```diff
@@ -0,0 +1,148 @@
+      how_support_equity: 'How do you support equity in your own classroom or role?',
+
+      expected_teacher_needs: clean_multiline(
+        'Teachers in Code.org’s Professional Learning Program join us with a wide range of experiences
+         (or lack thereof) in computer science education. What are some of the unique needs you’d expect
+         to find in a cohort of these teachers?'
+      ),
+      describe_adapting_lesson_plan:
+        'Describe a time when you’ve had to adapt a lesson plan in the moment to meet the needs of your students or participants.',
+
+      describe_strategies: clean_multiline(
+        'Have you used “lead learner” or “inquiry-based” strategies in your work with youth and/or adults?
+         If so, briefly describe how you have used these strategies.'
+      ),
+
+      example_how_used_feedback:
+        'Please provide a brief example of how you’ve used feedback __you’ve received from a colleague__ to improve your performance.',
+
+      example_how_provided_feedback:
+        'Please provide a brief example of how __you’ve provided feedback to a colleague__ and how that person responded to your feedback.',
+
+      hope_to_learn: 'What do you hope to learn from the facilitator development program?'
+    },
+
+    section_6_logistics: {
+      available_during_week:
+        'During the school year, are you available during the week (Monday - Friday) to attend phone calls or virtual meetings?',
+
+      weekly_availability: clean_multiline(
+        'During the school year, what times during the week (Monday - Friday)
+         are you available to attend phone calls or virtual meetings? (mark all that apply)'
+      ),
+
+      travel_distance: clean_multiline(
+        'What distance are you willing to travel to facilitate workshops?
+         Expenses may be covered for travel that requires overnight stays.'
+      )
+    },
+
+    section_7_submission: {
+      additional_info:
+        'Please provide any additional information you’d like Code.org to have about your application. (500 characters max)',
+
+      agree: 'By submitting this application, I agree to share my contact information and application with Code.org’s Regional Partners.'
+    }
+  }
+
+  LABEL_OVERRIDES = {
+    program: 'Please choose one curriculum for which you would like to become a facilitator this year.',
+    ability_to_meet_requirements: 'How would you rate your ability to meet the requirements and commitments for this program?',
+    grades_taught: 'What grade levels have you taught in the past?',
+    grades_currently_teaching: 'What grade levels do you teach?',
+    experience_leading: 'Which of the following do you have experience leading as a teacher?',
+    completed_pd: 'Which of the following Code.org professional learning programs did you complete as a teacher?'
+  }
+
+  ALL_LABELS = PAGE_LABELS.values.reduce(:merge)
+  ALL_LABELS_WITH_OVERRIDES = ALL_LABELS.map {|k, v| [k, LABEL_OVERRIDES[k] || v]}.to_h
+end
```

Comment:
Do we expect these to change much, or will they be relatively static?

In general, they feel more like a set of shared strings rather than shared constants. This is probably currently still the best way to share these, but I wonder if at some point we should be building a way to share strings that is different than sharing other constants.

Reply:
We don't expect them to change once the survey is out, but there's no guarantee. Agreed these are more like shared strings than constants, and I'm all for providing a different way to share them in the future... and also that this is currently the best way.